scripts/sagas/collection.js

```diff
@@ -39,13 +37,10 @@ export function* deleteAttachment(bid, cid, rid) {
 export function* listRecords(bid, cid, sort="-last_modified") {
   try {
     const coll = getCollection(bid, cid);
-    yield put(actions.collectionBusy(true));
```

Comment:
Note: I realized that dealing with these is much more easy to achieve in the reducers directly. That's how you learn I guess. Will need to do the same with other sagas.